src/Stepping.DbProviders.MongoDb/Stepping/DbProviders/MongoDb/MongoDbTransactionContext.cs

```diff
@@ -14,6 +14,6 @@ public MongoDbTransactionContext(MongoDbSteppingDbContext dbContext)
 
     public virtual async Task CommitAsync(CancellationToken cancellationToken = default)
     {
-        await ((MongoDbSteppingDbContext)DbContext).SessionHandle.CommitTransactionAsync(cancellationToken);
+        await ((MongoDbSteppingDbContext)DbContext).SessionHandle!.CommitTransactionAsync(cancellationToken);
```

Comment:
Why is it not `null`?
`MongoDbSteppingDbContextProvider` will create a `MongoDbSteppingDbContext` with null `SessionHandle`.

Reply:
I have changed it to check and throw.